### DISCUSSION TITLE
2021-11-10 Node Experience Team Update

### DISCUSSION THREAD

**Discussion by janewang** [near/community#123] [Node]
Node Experience team project board: https://github.com/orgs/near/projects/31
Attended: @mina86 @nikurt @janewang @bowenwang1996 @posvyatokum @marcelo-gonzalez

Q4 Board: https://github.com/orgs/near/projects/31

---

# Done
- @mina86 fixed gas accounting https://github.com/near/nearcore/pull/5184 https://github.com/near/nearcore/pull/5187
- @mina86 fixed a few of the nightly tests (https://github.com/near/nearcore/pull/5117, https://github.com/near/nearcore/pull/5122, https://github.com/near/nearcore/pull/5170)
- @mina86 fixed tests failing due to scipy weirdness (https://github.com/near/nearcore/pull/5158)
- @janewang hosted the November NVAB meeting, reviewing the roadmap, introduce metapool, and introduce the board charter

# In Progress
- @mina86 talked with @mm-near about taking over the rest (https://github.com/near/nearcore/issues/4618)
- @mina86 started looking into not verifying orphans only once https://github.com/near/nearcore/issues/3863
- @posvyatokum finishing work on adding access keys to fuzzer https://github.com/near/nearcore/issues/4374
- @posvyatokum writing a report on current code coverage of fuzzer https://github.com/near/nearcore/issues/4374
- @posvyatokum starting work on  ​client fuzzer https://github.com/near/nearcore/issues/4955
- @janewang is actively managing the seat price on testnet to onboard nodes (currently 64 nodes are on testnet), and is creating a technical score card on validators (readiness for mainnet). 
- @janewang is working with NF on delegation plan to onboard validators onto mainnet
- @janewang is building the chunk only producer pipeline with marketing ads and NFT drop
- @janewang is working on building a landing page for StakeWar III
- @marcelo-gonzalez put together a change to the indexer to collect historical validator info. Will be moving logic out to its own binary.
- @nikurt Working on monitoring the time nodes wait for all chunks to process a block https://github.com/near/nearcore/pull/5185
- @nikurt Staring at the logs of RPC nodes to understand why they fall out of sync https://github.com/near/nearcore/issues/5112
- @nikurt Has a plan for a more complete way of testing protocol upgrades on mocknet - upgrade nodes one by one.
- @nikurt Added structured output to state-viewer for investigating performance issues. https://github.com/near/nearcore/pull/5146

# Upcoming
- The next mainnet release for nearcore 1.22.0 is Monday, November 15 ⚙️

**Comment by maxzaver** [comment]
> @janewang hosted the November NVAB meeting, reviewing the roadmap, introduce metapool, and introduce the board charter

Would we great if we had some meeting notes to link to.

**Reply by janewang** [reply]
@nearmax  Sure, here's the link: https://nearprotocol.notion.site/NVAB-Meeting-of-November-9-2021-aaca4886331246c5aac74a6036a6df90
